projects/noloan/common/src/main/java/hasadna/noloan/common/SmsMessages.java

```diff
@@ -107,7 +107,7 @@ public void undoSuggestion(SmsMessage smsMessage) {
   }
 
   // Search Db/Inbox messages by sender & body. Return -1 if no message found
```

Comment:
Maybe `"// Search SmsMessages list by ... "` ?

Reply:
Didn't wrote it, don't care

Reply:
Yea I know :)

As part of your change of shared modules, you changed this function to work on general lists. 
I think that it'll be suitable to change the function's description relatively to your change, if you've changed its "goal".

Reply:
Nope, This is your's code.
The change I did did not approved.

Reply:
I actually meant the shared modules change. I think this had been approved no?
I'm referring to what's in the master

Anyway, I'm looking at it very simple:

If this function have been changed to be in a shared module for the Admin + NoLoan app
And functions as a function for general SmsMessage now...

I think `"// Search SmsMessages list by ..."`, describes it better.

That's all :)